src/openklant/components/klantinteracties/api/viewsets/klantcontacten.py

```diff
@@ -378,22 +380,76 @@ def perform_update(self, serializer):
         )
 
     @transaction.atomic
-    def perform_destroy(self, instance):
-        super().perform_destroy(instance)
-        token_auth = self.request.auth
+    def destroy(self, request, *args, **kwargs):
+        instance = self.get_object()
+        cascade = request.query_params.get("cascade", "false").lower() == "true"
```

Comment:
could you make sure this query parameter is documented in the API spec? You can do this with `from drf_spectacular.utils import OpenApiParameter`, defining a parameter and passing it to `extend_schema(..., parameters=[...])` (there's examples of this in Open Zaak as well)